splitio/engine/filters/base_filter.py

```diff
@@ -0,0 +1,28 @@
+import abc
+
+class BaseFilter(object, metaclass=abc.ABCMeta):
```

Comment:
rename to `Filter` please

Reply:
This is how interfaces named in the SDK code